packages/headless/src/composables/menuButton/createMenuButton.ts

```diff
@@ -0,0 +1,58 @@
+import { createId } from "../../utils/id";
+import { computed, ref } from "vue";
+
+import { createBuilder } from "../../utils/builder";
+
+export type MenuButtonValue = string | number | boolean;
+export type CreateMenuButtonOptions = { onSelect: (value: string) => void };
```

Comment:
@larsrickert I've added one, is ti okay?

Reply:
Yes but we should add the docs only to the `onSelect` instead of the whole type, so e.g.:
```ts
export type CreateMenuButtonOptions = {
  /**
   * Called when a menu item is selected (via mouse or keyboard).
   */
  onSelect: (value: string) => void;
};
```